### ISSUE TITLE
Text Field disabled when using prompt plugin

### ISSUE THREAD

**Issue by graftedbranch** [alexgorbatchev/jquery-textext#79]
I'm attempting to use the prompt plugin on a text field.  The form lives in a jquery dialog who's contents is loaded via ajax (not sure if that has anything to do with the issue or not).  I set the prompt on the field after the user takes some action on an adjacent grid (selects a grid row), so I'm setting the prompt in a method that gets called after some user interaction.

The prompt shows up, but I can't seem to get focus on the field by clicking on it and the prompt doesn't go away.  However if I navigate into the field via the tab key from an adjacent field, I can get focus and the prompt goes away, but the field is still disabled, meaning I can't type anything into the field.

Now this all is when I'm using the compressed version from the textextjs.com site (1.3.0).  However when I use the latest from github, the prompt works, I can gain focus, however, the prompt is off in alignment with the field.  This could just be a CSS issue.

I'm wondering if anyone has seen this and/or if you have any suggestions or ideas?

Thanks for the help.

**Comment by graftedbranch** [comment]
It looks like this is related to textext not supporting jquery UI?  After some investigating, I have found that it appears to be a CSS z-index issue, which makes sense since my form lives in a jquery-ui dialog, which I'm sure is setting a z-index.  I found that if I set the z-index on line 20 of textext.core.css to be 9999, then everything started working correctly.  Not sure what a 'valid' fix would be at the moment.
